### ISSUE TITLE
Add TypeScript

### ISSUE THREAD

**Issue by chwasifjameel** [chwasifjameel/react-conditional-rendering#17]
To ease the process for contributors and users we will be adding TypeScript. This will help the application perform better.

**Comment by 33kumarram** [comment]
Do you mean by replacing JavaScript code with TypeScript ?

**Comment by chwasifjameel** [comment]
Yes. I was planning to do so. Also, writing interfaces so that it can stop giving errors in production
